Failed to build using 4.20 kernel

hey guys,
kernel: 4.20.0-arch1-1-ARCH 

Everything worked just fine until now. Faced no issues through the past builds. 

the last lines of the make log file 

> /var/lib/dkms/rtl8723de/5.1.1.8_21285.20171026_COEX20170111-1414/build/make.log

```
/var/lib/dkms/rtl8723de/5.1.1.8_21285.20171026_COEX20170111-1414/build/os_dep/linux/ioctl_cfg80211.c: In function ‘rtw_get_systime_us’:
 /var/lib/dkms/rtl8723de/5.1.1.8_21285.20171026_COEX20170111-1414/build/os_dep/linux/ioctl_cfg80211.c:352:2: error: implicit declaration of function ‘get_monotonic_boottime’; did you mean ‘getboottime’? [-Werror=implicit-function-declaration]
   get_monotonic_boottime(&ts);
   ^~~~~~~~~~~~~~~~~~~~~~
   getboottime
 cc1: some warnings being treated as errors
 make[2]: *** [scripts/Makefile.build:292: /var/lib/dkms/rtl8723de/5.1.1.8_21285.20171026_COEX20170111-1414/build/os_dep/linux/ioctl_cfg80211.o] Error 1
 make[1]: *** [Makefile:1563: _module_/var/lib/dkms/rtl8723de/5.1.1.8_21285.20171026_COEX20170111-1414/build] Error 2
 make[1]: Leaving directory '/usr/lib/modules/4.20.0-arch1-1-ARCH/build'
 make: *** [Makefile:1885: modules] Error 2
```

--- a/os_dep/linux/ioctl_cfg80211.c
+++ b/os_dep/linux/ioctl_cfg80211.c
@@ -349,7 +349,13 @@ static u64 rtw_get_systime_us(void)
 {
 #if (LINUX_VERSION_CODE >= KERNEL_VERSION(2, 6, 39))
 	struct timespec ts;
-	get_monotonic_boottime(&ts);
+
+	/* get_monotonic_boottime() deprecated in linux kernel 4.20 */
+	// get_monotonic_boottime(&ts);
+
+	/* Corresponding functionality to deprecated function with refernece to linux kernel source */ 
+	ts = ktime_to_timespec(ktime_get_boottime());
+
 	return ((u64)ts.tv_sec * 1000000) + ts.tv_nsec / 1000;
 #else
 	struct timeval tv;

